README.md

```diff
@@ -152,12 +152,18 @@ For å trigge **manuell deploy** kan du gå til `Actions -> (velg workflow) -> R
 ## Lokal bygg/test - docker
 
 Docker må være installert og kjørende for at lokal bygg/test skal fungere. Hvis man bruker docker desktop så klarer testene
-å finne docker socket automatisk. Hvis man har colima kjørende så må man ha en symlink:
+å finne docker socket automatisk. Hvis man har colima kjørende så må man sette opp noen miljøvariabler.
 
 ```
-sudo ln -sf $HOME/.colima/default/docker.sock /var/run/docker.sock
+DOCKER_HOST=unix://${HOME}/.colima/default/docker.sock
+TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE=/var/run/docker.sock
+TESTCONTAINERS_RYUK_DISABLED=true
```

Comment:
Denne gjør ingen forskjell for meg tilsynelatende, men hvorfor skal vi ikke ha den der. Den sletter containers/networks/volumes/images  når testen er ferdig, kjipt om vi må gjøre det selv men ikke vet grunnen pga det her? :D

Reply:
Tatt rått fra https://nav-it.slack.com/archives/C60FFACN5/p1675063655436059?thread_ts=1675063497.442989&cid=C60FFACN5

Reply:
Skal ikke være nødvendig å skru av Ryuk. Hos meg har jeg kun følgende: 

```
export TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE=/var/run/docker.sock
export DOCKER_HOST="unix://${HOME}/.colima/docker.sock"
```

Reply:
Samme som @Watercolours  her.

Reply:
Mulig at det siste der er spesifikk til apple M chipset and/or oracle (ser nå at det nevnes i starten av tråden).

Gitt at vi ikke bruker oracle - en sist sjekk - noen her med M1 eller M2 chipset på mac'en sin?